### ISSUE TITLE
Mass unsubscribe and delete not working in Backend

### ISSUE THREAD

**Issue by michi-p** [vincent2090311/magento2-campaignmonitor#13]
When using the Magento Backend to unsubscribe or delete newsletter subscriptions the following error occurs:
```
`Invalid argument supplied for foreach() in vendor/luma/module-campaignmonitor/Observer/MassUnsubscribeUsers.php on line 48`
```

You need to change the affected line from
```php
foreach ($subscribers as $id) {
```
to
```php
foreach (\explode(',', $subscribers) as $id) {
```
as `$subscribers` is a string. When multiple opitons are selected it is a comma-separated string.

**Comment by vincent2090311** [comment]
Thanks @michi-p 
I've updated new code to master. Im going refactor whole module and publish very soon

**Comment by vincent2090311** [comment]
@michi-p 
The issue is already fixed in v1.0.3
